### ISSUE TITLE
interactive CLI has now quit or exit command.

### ISSUE THREAD

**Issue by vassudanagunta** [raineorshine/npm-check-updates#1414]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)

---

Currently the only way, AFAIK, is to issue a terminal kill command.

Ideally `q`  or `x` to exit with nothing upgraded or changed.

If agreed, I can take a stab at a PR when I get some free time.

**Comment by raineorshine** [comment]
Thanks for the suggestion. I'd be happy with `q` for quit.

**Comment by ZaLiTHkA** [comment]
just thinking out loud here, but perhaps this could be "fixed" by simply updating the input desrciptions for interactive mode?

pressing "escape" cancels the interactive mode prompt, returning the user to their terminal, as (I imagine) the OP would expect `q` or `x` to do.

**Comment by vassudanagunta** [comment]
@ZaLiTHkA Good to know. But not very obvious or intuitive: In my experience, `ESC` is a standard idiom for cancellation in GUI apps. As far as I can remember, all the CLIs I've used exit on `q` or `x`. Either works for me, but I would suggest that it match most user expectations / CLI conventions.

**Comment by ZaLiTHkA** [comment]
@vassudanagunta I agree with you on that point, it's definitely not very "intuitive" as it is right now.

on the other hand, the fact that there is already a mechanism in place to "cancel" interactive mode also means that adding support for a `q` or `x` shortcut to do the same thing _shouldn't_ need entirely new functionality, just a small adjustment to which key actually triggers it.

either way, a short message in the initial usage notes would clear this up for everyone.
